dockerclient/client.go

```diff
@@ -685,6 +685,11 @@ func (e *ClientExecutor) EnsureContainerPath(path string) error {
 	return e.createOrReplaceContainerPathWithOwner(path, 0, 0)
 }
 
+func (e *ClientExecutor) EnsureContainerPathAs(path, user string, mode *os.FileMode) error {
+	// FIXME: user and mode are currently ignored.
+	return e.createOrReplaceContainerPathWithOwner(path, 0, 0)
```

Comment:
Must not be merged before this is resolved.

Reply:
I've not implemented it because we would have to convert the userspec into UID:GID.

Do you have any suggestions?  Do we need to copy the `Builder.userForCopy` function from buildah?

Reply:
You might refactor parts of the dockerclient `CopyContainer()` method, which has to do similar work to support a `--chown` flag.